specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/Jobs.json

```diff
@@ -430,9 +450,136 @@
         },
         "x-ms-examples": {
           "Terminate a Container Apps Job": {
-            "$ref": "./examples/Job_Stop.json"
+            "$ref": "./examples/Job__Stop_Execution.json"
```

Comment:
>__ [](http://example.com/codeflow?start=35&length=2)

typo? (double underscore)

Reply:
@vinisoto yes, it was a typo.. fixed it.